src/converter-gen/Makefile

```diff
@@ -2,15 +2,29 @@ include ../testing/make_common.mk
 
 # XML Calabash prefers absolute paths
 test_metaschema:=../testing/models_metaschema.xml
-output_folder:=test_output
+module_path:=$(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
+output_folder:=$(module_path)/test_output
+
+xspec_script=$(realpath $(module_path)/../../support/xslt3-functions/xspec-dev/mvn-saxon-xspec-batch-quiet.sh)
+xspec_ci_script=$(realpath $(module_path)/../../support/xslt3-functions/xspec-dev/mvn-saxon-xspec-batch.sh)
+folder=.
 
 .PHONY: test
 test: smoke-test ## Run all tests
 
 .PHONY: smoke-test
-smoke-test: ## Run all smoke-tests
+smoke-test: ## Apply generators to ../testing/models_metaschema.xml metaschema
 	./mvn-converters-xpl.sh "$(test_metaschema)" $(output_folder) models_metaschema
 
+.PHONY: spec-test
+spec-test: ## Run all XSpec in testing folder
+	LOGFILE="$(output_folder)/converter-gen-xspec.log" $(xspec_ci_script) \
+		"folder=$(module_path)/testing/" \
+		"report-to=$(output_folder)/converter-gen_report.html" \
+		"junit-to=$(output_folder)/converter-gen_junit-report.xml" \
+		"stop-on-error=yes" \
+		"recurse=yes"
```

Comment:
Minor, non-blocking: in the future it might be good to set Makefile variables at the top of this file for `stop-on-error` and/or `recurse` to override the default.

Reply:
https://github.com/usnistgov/metaschema-xslt/issues/121